README.md

```diff
@@ -60,9 +60,10 @@ INFO: Registered handler to forward  attachmentActions:created events
 If the following environment variables are set:
 
 * TOKEN
+* TARGET
 * PORT
 
-hookbuster will automatically register for all of the [webhook firehose](https://developer.webex.com/docs/api/guides/webhooks/the-firehose-webhook) events, using the TOKEN and then forward any received events to localhost:PORT
+hookbuster will automatically register for all of the [webhook firehose](https://developer.webex.com/docs/api/guides/webhooks/the-firehose-webhook) events, using the TOKEN and then forward any received events to TARGET:PORT. If TARGET is not set it defaults to `localhost`
```

Comment:
It does not default to localhost, it prompts you to enter a target. Can we have this corrected please?

Reply:
@legion49f we could change the doc to something like this: 
"hookbuster will automatically register for all of the [webhook firehose](https://developer.webex.com/docs/api/guides/webhooks/the-firehose-webhook) events, using the TOKEN and then forward any received events to TARGET:PORT. If you only set PORT and TOKEN in your environmental variables, then TARGET will be defaulted to `localhost`. Otherwise (if you do not use environmental variables) you will be prompted to enter a target manually."

Reply:
LGTM